### PR TITLE
[PRD-4160] Xul based DrillDown dialogs in Report Designer cannot be internationalized

### PR DESCRIPTION
**Pull request by mr9d**
- Missed Target and Tooltip fields

### CODE REVIEW

File: designer/report-designer-extension-pentaho/source/org/pentaho/reporting/designer/extensions/pentaho/drilldown/swing/SwingRemoteDrillDownUi.java [hunk 1]
```diff
@@ -56,6 +61,15 @@
   /** Top=level components container. */
   private JPanel editor;
 
+  /** Server URL input field. */
+  JTextField serverUrlField;
```
Comment (tmorgner) [inline designer/report-designer-extension-pentaho/source/org/pentaho/reporting/designer/extensions/pentaho/drilldown/swing/SwingRemoteDrillDownUi.java]: Is there a specific reason why these fields are package protected? Shouldn't they be private instead? If you needed to access them or their contents, accessors would do that job more safely.

Comment (mr9d) [inline designer/report-designer-extension-pentaho/source/org/pentaho/reporting/designer/extensions/pentaho/drilldown/swing/SwingRemoteDrillDownUi.java]: Nope, I just missed it. Fixed, thank you.

**Comment by mr9d** [comment]
@pamval could you please rerun Wingman for this PR?

**Comment by tmorgner** [comment]
@wingman-pentaho is lazy, that bugger only monitors the engine/core project, not the other modules. But then again, poor Wingman is locked away in an ice-cold server room with no sunlight, so no wonder he's not willing to look at side projects. I would neither if I were about to freeze to death :)

**Comment by mr9d** [comment]
Very comprehensible explanation =)

**Pull request merged**